documents/markdown/how-to-auth.md

```diff
@@ -165,7 +165,7 @@ for example:
 
 ```
 curl https://on-stage.copyrighthub.org/v1/onboarding
-{"status": 200, "data": {"service_name": "Open Permissions Platform Onboarding Service", "hub_id": "hub1", "version": "1.0.0", "default_resolver_id": "copyrighthub.org", "service_id": "5e7c2be8f9f8dc8456f61db81e004d32"}}
+{"status": 200, "data": {"service_name": "Open Permissions Platform Onboarding Service", "hub_id": "hub1", "version": "1.0.0", "default_resolver_id": "openpermissions.org", "service_id": "5e7c2be8f9f8dc8456f61db81e004d32"}}
```

Comment:
For this example the default resolver id would still be copyrighthub.org wouldn't it?